src/main/scala/encry/view/state/UtxoState.scala

```diff
@@ -65,7 +66,7 @@ class UtxoState(override val persistentProver: PersistentBatchAVLProver[Digest32
     val totalFees: Amount = regularTransactions.map(_.fee).sum
 
     val regularApplyTry: Try[Unit] = applyTry(regularTransactions)
-    val coinbaseApplyTry: Try[Unit] = applyTry(Seq(coinbase), totalFees)
+    val coinbaseApplyTry: Try[Unit] = applyTry(Seq(coinbase), totalFees + EncrySupplyController.supplyAt(height))
```

Comment:
Do you think that using just `Double` inside of `supplyAt(height: Height): Amount` is a gapped decision?